.percy.js

```diff
@@ -0,0 +1,15 @@
+const { executablePath } = require('puppeteer')
+
+/**
+ * Percy config
+ *
+ * @type {import('@percy/config').PercyConfigObject}
+ */
+module.exports = {
```

Comment:
**nitpick** Looks like percy accepts `percy.config.js` as a config file, it would align with the format of most our config files so I'd be keen to prefer that (especially as they didn't go for the `.<PROJECT_NAME>rc.js` convention and just `.<PROJECT_NAME>.js`) 😊

Reply:
✅ Done